index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>CAkE Portfolio</title>
+</head>
+
+<body>
+    <main class="main">
+
+        <header class="header">
+            <a href="#" class="my-logo">CAkE</a>
+            <nav class="nav-bar">
+
+
+                <span class="mob-menu"><img src="Images&Icons/Enabled.png" alt=""></span>
+            </nav>
+        </header>
+
+        <section class="banner">
+            <!-- <img class="banner-img" src="Images&Icons/mobile shapes.png" alt=""> -->
+            <div class="intro-text">
+                <h1>I'm CAkE <br>
+                    Glad to see <br> you!
+                </h1>
+                <p>i'm software developer! i can help you build a product, feature or website look through
+                    some of my work and experience! if you like what you see and have a project you need
+                    coded, don't hesitate to contact me. </p>
+            </div>

```

Comment:
- [OPTIONAL] I think it would be better if you added horizontal padding or margins between `<h1>` tag and `<p>` tag 👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/176800501-4fc967a7-a5cf-47f8-9c39-3b1e9b718432.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/176800590-e408d559-93dc-48fd-99d8-7db5e1aaaac7.png" width="500" /> |